src/components/Shade.jsx

```diff
@@ -9,7 +10,7 @@ const Shade = ({
     setShadeRef,
 }) => (
     <div
-        className={`shade ${className || ''} ${visible ? '' : 'hidden'}`}
+        className={classNames('shade', className || '', { hidden: !visible })}
```

Comment:
```suggestion
        className={classNames('shade', className, { hidden: !visible })}
```

`classNames` is capable of handling `whitespaces` and `undefined` `classNames`.

Reply:
good point. thanks